specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/examples/RuleSet_Create.json

```diff
@@ -0,0 +1,70 @@
+{
+    "parameters": {
+      "api-version": "2020-09-01",
+      "subscriptionId": "subid",
+      "resourceGroupName": "RG",
+      "profileName": "profile1",
+      "ruleSetName" : "ruleSet1",
+      "ruleSet" : {
+        "properties": {
+            "rules": [{
+                "id": "/subscriptions/subid/resourceGroups/RG/providers/Microsoft.Cdn/profiles/profile1/ruleSets/ruleSet1/rules/rule1"
```

Comment:
>ruleSet1 [](start = 113, length = 8)

For creation of ruleSet1 itself, can ruleSet1 be referenced in "id" here *before* it got created?
For a user wanting to create new ruleset with few rules in it, what is the flow? Does the ruleset get created with 0 rules first and then it uses ruleset reference to create rules under it?